spec/support/bb_client_helpers.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module BB
+  module ClientHelpers
+    TOKEN = 'GkuX2OZ4dCE=48xrH6ObGXZ45ZAg70LBahi7CjswZe8SZGKMUVFIU88='
+
+    def authenticated_client
+      BB::Client.new(session: { user_id: 123, expires_at: Time.current + 60 * 60, token: TOKEN })
+    end
+  end
+end
```

Comment:
Comment says client helper not needed but I still see class here and ref to it in spec above. Is that accidental or am I misunderstanding what's not needed?